src/patterns/molecules/heading/heading.scss

```diff
@@ -1,5 +1,26 @@
 @use "../../../shared-styles/normalize"; // Always used first
+@use "../../../shared-styles/baseline";
 @use "../../../shared-styles/link";
+@use "../../../shared-styles/space";
+
+@mixin _default-block-spacing {
+  @include space.block($end: space.calculate($block-size: baseline.for-units(4), $unit-size: baseline.for-units(8)));
+}
+
+h1 {
```

Comment:
Unlike the typography, there shouldn't be any need to reuse the heading-specific baseline grid outside of an actual heading. Even if there does turn out to be, this heading emulation should probably define its own spacing rather than using the headings ones.

Reply:
It's safer to only ever use bare element selectors in the normalize file. In the event there's a heading that doesn't use the pattern, its spacing changes whether this file is included or not...

Reply:
Agree. For simplicity will go down a similar route as for the typography. Was worth trying something else, but it doesn't work.